release-notes/opensearch-dashboards.release-notes-2.0.0.md

```diff
@@ -0,0 +1,96 @@
+## Version 2.0.0 Release Notes
```

Comment:
There are some changes like this from other commits that shouldn't be in the PR diff. Did something happen with the rebase?

Reply:
Fixed the rebase.